components/live/header.tsx

```diff
@@ -26,8 +28,13 @@ export const LiveHeader = (props: LiveHeaderprops) => {
               "flex flex-row items-center max-md:justify-center"
             )}
           ></div>
+          <h1
+            dangerouslySetInnerHTML={{
+              __html: sanitiseXSS(title, spanWhitelist) || "",
+            }}
+          ></h1>
```

Comment:
Can you make this bold like the old one, and the same size?

Reply:
Done